wikipedia-sentences-query-while-indexing/README.md

```diff
@@ -83,60 +78,55 @@ In order to run the example you will need to do the following:
 
 If you want to use the entire dataset, run `bash get_data.sh` and then modify the `DATA_FILE` constant (in `app.py`) to point to that file.
 
 ### 🏃 Step 3. Running the Flows
 
-1. In one terminal session, run the command `jinad`.
+In this example, we use [JinaD]((https://docs.jina.ai/advanced/daemon/#remote-management-via-jinad)) to serve the two Flows (Index and Query) and listen to incoming requests.
 
-    `JinaD` is our platform for running Jina services (Flows, Pods) remotely, wherever you want to run them. In this example, we use `JinaD` to serve the two Flows (Index and Query). You can read more about it [in the docs](https://docs.jina.ai/chapters/remote/jinad).
+1. Start `JinaD` server using the below command.
 
-2. In another terminal, run `python app.py -t flows`
+   ```bash
+   docker run --add-host host.docker.internal:host-gateway \
+           -v /var/run/docker.sock:/var/run/docker.sock \
+           -v /tmp/jinad:/tmp/jinad \
+           -p 8000:8000 \
+           --name jinad \
+           -d jinaai/jina:master-daemon
+   ```
 
-    This will create the two Flows, and then repeatedly do the following, every 20 seconds:
+2. Run `python app.py -t flows`
 
-    1. Index 5 Documents
-    2. Send a `DUMP` request to the DBMS (Index) Flow to dump its data to a specific location
-    3. Send a `ROLLING_UPDATE` request to the Query Flow to take down its Indexers and start them again, with the new data located at the respective path
-    
-    **Notice** the logs of the operations.
+    This will create the two Flows, and then repeatedly do the following (which can also be done in any other REST client), every 10 seconds:
 
-    **Warning**: the data file is limited to 200 documents. Once that is exhausted, the process will terminate. If you want to use the entire dataset, run `bash get_data.sh` and then modify the `DATA_FILE` constant to point to that file.
+    1. Index 5 Documents.
+    2. Send a `DUMP` request to the DBMS (Index) Flow to dump its data to a specific location.
+    3. Send a `ROLLING_UPDATE` request to the Query Flow to take down its Indexers and start them again, with the new data located at the respective path.
+
+    **Warning**: If you want to use the entire wikipedia dataset, run `bash get_data.sh` and then modify the `DATA_FILE` constant to point to that file.
 
 ### 🔎 Step 4: Query your data
 
-Finally, in a third terminal, run `python app.py -t client`
+Finally, in a second terminal, run `python app.py -t client`
 
 This will prompt you for a query, send the query to the Query Flow, and then show you the results.
 
-**Notice** how the number of total matches grows as step **2** from above gets repeated.
+Since the Flows uses `http` protocol, you can query the REST API with whatever `Client` provided within jina or use `cURL`, `Postman` or [custom Swagger UI provided with jina](https://docs.jina.ai/fundamentals/practice-your-learning/#query-via-swaggerui) etc.
```

Comment:
Maybe still add a `cURL` example here

Reply:
`Client` should be used for python-based based requests. We can include the `cURL` syntax in the docs

Reply:
We should still show it here. Or link to the REST API docs, so that the user can tell how the REST call would need to look like. The Client is pretty opaque at first glance.